src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: My Open Source Journey @Layer5
+subtitle: From Contributor to SWE Intern
+date: 2022-01-14 10:30:05 -0530
+author: Gaurav Chadha
+thumbnail: ./profile-banner.png
+description: "My journey as an Open Source contributor at Layer5. How I started from scratch, got involved in this community
+and gained many new experiences."
+type: Blog
+category: Community
+tags:
+  - Open Source
+  - Experience
+  - Layer5
+published: true
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+import Profile from "./gaurav-chadha.png"
+
+<BlogWrapper>
+<img src={Profile} className="profileImg" align="right" alt="profile-img" />
+<p>
+  Hey👋, I am Gaurav Chadha, a student from New Delhi, India with lots of dreams, ambitions, and zeal to learn and grow more each and every day.
+</p>
+<h2 id="journey">How did it all begin?</h2>
+<p>
+My journey in open source started with late-night surfing on LinkedIn.
+I saw a post about the Layer5 organization and open source which drove me to their website and{" "}<a href = "https://slack.layer5.io">Slack workspace</a>. And then, 💥 a single-click was all it took to kickstart my journey in Open Source and set in-motion a new chapter in my career development and relationships with new colleagues. 
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+Despite the warm welcome that I receive upon joining Layer5's Slack workspace, I was a bit shy to introduce myself in the{" "}<a href = "https://layer5io.slack.com/archives/C019426UBNY">#newcomers channel</a>, but then I saw so many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
+The best part for newcomers in Layer5 is that not only is each individual warmly welcomed, but that
+Layer5 {" "} <a href = "https://layer5.io/community/meshmates">[MeshMates]</a> and Community Managers host a weekly meeting (see the{" "}<a href = "https://layer5.io/community/calendar">[community calendar]</a>) dedicated to the success of newcomers, helping each person find a foothold in the community. The overview provided in each weekly Newcomers meeting gives you the leg up that you need to get started with any of{" "} <a href = "https://layer5.io/projects">Layer5's projects</a> and orients you to the how the community works (the{" "} <a href = "https://layer5.io/community/handbook">community handbook</a> is an excellent resource in this regard) and who to contact in your areas of interest.
```

Comment:
```suggestion
Layer5 {" "} <a href = "https://layer5.io/community/meshmates">[MeshMates]</a> and Community Managers host a weekly meeting (see the{" "}<a href = "https://layer5.io/community/calendar">community calendar</a>) dedicated to the success of newcomers, helping each person find a foothold in the community. The overview provided in each weekly Newcomers meeting gives you the leg up that you need to get started with any of{" "} <a href = "https://layer5.io/projects">Layer5's projects</a> and orients you to the how the community works (the{" "} <a href = "https://layer5.io/community/handbook">community handbook</a> is an excellent resource in this regard) and who to contact in your areas of interest.
```

Reply:
Actually... this should be a `<Link>`, yeah?